translate/src/modules/machinery/components/MachineryTranslation.tsx

```diff
@@ -52,6 +53,8 @@ export function MachineryTranslationComponent({
       if (llmTranslation) {
         logUXAction('LLM Translation Copied', 'LLM Feature Adoption', {
           action: 'Copy LLM Translation',
+          localeCode: locale.code,
+          targetLanguage: locale.name,
```

Comment:
I wouldn't include `targetLanguage`, because `localeCode` is already unambiguous. It's also better to keep both related UX actions that are logged consistent in that regard.